sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -753,7 +753,8 @@ public ServiceBusReceiverAsyncClient buildAsyncClient() {
          *     queueName()} or {@link #topicName(String) topicName()}, respectively.
          */
         public ServiceBusReceiverClient buildClient() {
-            return new ServiceBusReceiverClient(buildAsyncClient(), retryOptions.getTryTimeout());
+            return new ServiceBusReceiverClient(buildAsyncClient(), retryOptions.getTryTimeout(),
+                ServiceBusConstants.SHORT_TIMEOUT_BETWEEN_MESSAGES);
```

Comment:
I'm not sure why this is passed in as a parameter. It seems very specific to a type of algorithm.